core/templates/domain/improvements/task-entry.model.ts

```diff
@@ -38,7 +38,7 @@ export interface TaskEntryBackendDict<TaskType = string> {
   'status': string;
   'resolver_username': string;
   'resolver_profile_picture_data_url': string;
-  'resolved_on_msecs': number;
+  'resolved_on_msecs'?: number;
```

Comment:
Why is optional added here?

Reply:
Optional was added here because I wanted to assign undefined to a variable of type number.

Reply:
as you told me to use null instead of undefined, I thought discussing the ? is not necessary, as I am going to remove it.